src/instance/discord/chat-manager.ts

```diff
@@ -48,6 +48,7 @@ export default class ChatManager extends EventHandler<DiscordInstance> {
       instanceType: InstanceType.DISCORD,
       channelType: channelType,
       channelId: event.channel.id,
+      userId: Number.parseInt(event.author.id),
```

Comment:
Don't convert the id to other format. If it is `string`, then change the `userId` to be `string | undefined` instead of `number | undefined`.